<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="treeView2"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="buttonStart1Doc">
        <property name="text">
         <string>StartDocument</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonEndDoc1">
        <property name="text">
         <string>EndDocument</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStartEl">
        <property name="text">
         <string>StartElement</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonEndEl1">
        <property name="text">
         <string>EndElement</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpenFile"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Окно</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu1"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_4">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionqwe">
   <property name="text">
    <string>qwe</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
